lib/about.js

```diff
@@ -17,7 +17,8 @@ module.exports = function(opts) {
 				description: name,
 				support: 'next.team@ft.com',
 				supportStatus: 'active',
-				appVersion: commit
+				appVersion: commit,
+				deployTime: new Date().toString()
```

Comment:
`new Date().toISOString()` is more parseable by future systems that might use this data